### PR TITLE
chore(trading-competition): Remove promotional LeadInBanner from page headers

### PR DESCRIPTION
**Pull request by ChefHutch**
Title says it all. Just removing the LeadInBanner.

We can do something nicer with a PromotionalBanner component when we have a plan for how to use the banners going forward

### CODE REVIEW

**Review by Chef-Chungus — APPROVED** [review]

File: src/views/Home/Home.tsx [hunk 1]
```diff
@@ -81,32 +80,29 @@ const Home: React.FC = () => {
   const TranslateString = useI18n()
 
   return (
-    <>
-      <LeadInBanner />
```
Comment (ChefHutch) [inline src/views/Home/Home.tsx:85]: Some fairly agressive diffs in here. In each instance the `<LeadInBanner>` is being removed as well as the wrapping fragment

**Comment by memoyil** [comment]
LeadInBanner can stay, but we can empty it for future uses therefore we will not have to add it again in the future.

**Review by RabbitDoge — CHANGES_REQUESTED** [review]
If the component is not used anymore, please remove it

**Comment by RabbitDoge** [comment]
#910

**Pull request closed without merging**